src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -0,0 +1,69 @@
+$(function () {
+  function sendEvent(eventCategory, eventAction, eventLabel) {
+    if (!eventCategory) {
+      throw new Error('Event category is required')
+    }
+    if (!eventAction) {
+      throw new Error('Event action is required')
+    }
+    if (!eventLabel) {
+      throw new Error('Event label is required')
+    }
+
+    ga('send', 'event', eventCategory, eventAction, eventLabel)
+  }
+
+  function labelText(labelElement) {
+    if (!labelElement) {
+      throw new Error('Label element is required')
+    }
+
+    for (var content of labelElement.contents()) {
+      const text = $(content).text().trim()
+      if (text && content.class !== 'form-hint') {
+        return text
+      }
+    }
+    return undefined
+  }
+
+  function formValue(formElement, inputName) {
+    if (!formElement) {
+      throw new Error('Form element is required')
+    }
+    if (!inputName) {
+      throw new Error('Input name is required')
+    }
+
+    var inputElement = formElement.find('input[name=' + inputName + ']')
+    if (inputElement.length > 0) {
+      switch (inputElement[0].type) {
+        case 'radio':
+          if (!inputElement.is(':checked')) {
+            return undefined
```

Comment:
This will propagate up to `sendEvent` function and write errors in JavaScript console. Nut a big problem but not too elegant as well I think.

Reply:
Agree, would be better not to throw errors when radio button is not selected.